packages/website/pages/files.js

```diff
@@ -302,7 +321,19 @@ export default function Files({ user }) {
           <When condition={status !== 'loading'}>
             <>
               <div className="flex items-center mb3">
-                <h1 className="flex-auto chicagoflf mv4">Files</h1>
+                <div className="flex-auto chicagoflf mv3">
+                  <h1>Files</h1>
+                  <label className="webViewer-toggle mt1">
+                    <input
+                      type="checkbox"
+                      name="toggleWebViewer"
+                      className="mr1"
+                      onChange={(e) => handleWebViewerToggle(e)}
+                    />
+                    Use IPFS web viewer instead of IPFS applications where
```

Comment:
I'm not sure what "web viewer" means!? It means open on the gateway in your browser?

Reply:
Yes, not sure on the correct wording here. Open to your input...just felt important to have an opt-in for users that want to use https://ipfs.io as the viewer.

Reply:
@alanshaw is the concern the label used, or the underlying approach of giving user the choice?